_tests_/testDataHelpers.ts

```diff
@@ -0,0 +1,107 @@
+import cql from 'cql-execution';
```

Comment:
This file should be moved to a different branch for the follow on work to run against the fixture data.